config/locales/en.yml

```diff
@@ -305,18 +314,18 @@ en:
       hint: If some don't apply, enter 0.
       adult_dependants:
         label: Adult dependants
-        hint: For example, a parent, grandparent, aunt or uncle.
+        hint: Any adult relative who lives in the same household and is financially dependant on your client.
         instruction: Enter number of adult dependants
       child_dependants:
         label: Child dependants
-        hint: For example, your client's child, grandchild, niece or nephew.
+        hint: For example your client’s children or another child relative who lives in their household. Do not include any foster children.
         instruction: Enter number of child dependants
     employment:
       caption: Your client's finances
       heading: Add your client's salary breakdown
       second_caption: You can find this on your client's payslips.
       gross_income:
-        input: Employment income
+        input: Gross Employment Income
```

Comment:
I know the Google doc has these all capitalised, but I think that may not have been in the conscious awareness that that would be a convention-breaking change. I am 99% certain that Chris would confirm that, for consistency, this should be 'Gross employment income' if asked.